website/content/docs/agent/config/config-files.mdx

```diff
@@ -736,6 +736,13 @@ Refer to the [formatting specification](https://golang.org/pkg/time/#ParseDurati
 
 - `server` Equivalent to the [`-server` command-line flag](/consul/docs/agent/config/cli-flags#_server).
 
+- `server_rejoin_age_max` controls the allowed maximum age of a stale server attempting to rejoin a cluster.
+  E.g., A user attempts to start a previously running server with the following last seen Unix timestamp `1672531200` (2023-Jan-01 00:00:00)
+  saved in the `server_metadata.json` file in the configured data directory.
```

Comment:
Would it be clearer not to refer to these internal details and instead say something like:

> If a server is not running for this period, then it will refuse to start up again until an operator intervenes. This is to protect clusters from instability caused by decommissioned servers accidentally being started again.